### PR TITLE
Add count (number value) to facets

### PR DESCRIPTION
**Pull request by ThibodeauJF**
screen shot 2018-02-28 at 4 44 11 pm

Facets are more a little bit akin to search ui, facet name that are too long get an ellipsis at the end and the count is now available. Had to refactor the style a bit to make them all fit on the same line.

### CODE REVIEW

File: scss/components/facet.scss [hunk 1]
```diff
@@ -17,173 +16,187 @@
...
+    clear: both;
+  
+    background: $white;
+    border-top-right-radius: $border-radius;
+    border-top-left-radius: $border-radius;
+
+    .facet-header-eraser {
+      position: absolute;
+      top: 0;
+      right: 25px;
```
Comment (MichelML) [inline scss/components/facet.scss]: i know the code base region you're in was not a good example for what i'm about to say, but we have a rule in vapor which is to have all `px` and `%` values in variables, this way if you need to make calculations based on these values, you can reuse them. Not needed for `translateY(-50%);` but for everything else it should be done

File: scss/components/facet.scss [hunk 2]
```diff
@@ -17,173 +16,187 @@
...
+    max-height: $facet-more-height;
+    padding: $dropdown-choices-top-bottom-margin 0;
+    margin: 3px 0;
+    overflow-x: hidden;
+    overflow-y: auto;
+  
+    list-style: none;
+  
+    background-color: $pure-white;
+    border: 1px solid $medium-grey;
```
Comment (MichelML) [inline scss/components/facet.scss]: you can use `mod-border` IIRC

**Comment by MichelML** [comment]
@ThibodeauJF my previous comments have been outdated with recent updates, but they might still apply

File: scss/components/facet.scss [hunk 3]
```diff
@@ -17,173 +17,189 @@
...
   border-radius: $border-radius;
-}
 
-.facet-header {
-  position: relative;
+  &:last-child {
+    &.facet-opened {
+      .facet-search-results {
+        top: auto;
+        bottom: 60px;
```
Comment (sroy3) [inline scss/components/facet.scss]: This should be a variable

Comment (sroy3) [reply scss/components/facet.scss]: All pixel values in this file should all be variables.

**Comment by ThibodeauJF** [comment]
Values now in variables

**Pull request merged**